crates/ee-acct-runtime/src/exec_processing.rs

```diff
@@ -0,0 +1,106 @@
+//! Procedures relating more specifically to execution processing.
+
+use strata_ee_acct_types::{
+    CommitBlockData, CommitCoinput, CommitMsgData, EeAccountState, EnvError, EnvResult,
+    ExecutionEnvironment,
+};
+use strata_ee_chain_types::ExecBlockNotpackage;
+
+use crate::verification_state::UpdateVerificationState;
+
+/// Verifies a commit against the account's current state using a coinput which
+/// reveals all the EE blocks and notpackage data, accumulating effects in the
+/// verification state.
+pub fn verify_commit<E: ExecutionEnvironment>(
+    vstate: &mut UpdateVerificationState,
+    astate: &EeAccountState,
+    commit: &CommitMsgData,
+    coinp: &CommitCoinput,
+    ee: &E,
```

Comment:
Shouldn't we be using something similar to the `strata-predicate` here?
I'm fine if this implies as leaving this as a TODO.